client/layout/guided-tours/docs/DEBUGGING.md

```diff
@@ -11,7 +11,13 @@ In the following example, the tours `gdocsIntegrationTour` and `editorInsertMenu
 
 ![Tour seen](./img/tour-seen.png)
 
-Currently, there is no systematic way of clearing a user's history for debugging. One way you can clear it is by applying the [`clean-history.patch`](./patch/clean-history.patch) to your working tree, load the page to clean the history, and remove it again.
+To clear a user's tour history, simply visit Calypso with `tour=reset` as a query argument. This will then call `resetGuidedToursHistory`:
```

Comment:
Doesn't it also require to clear the application browser state? I couldn't trigger a tour I've had seen by using `tour=reset` until I delete my local application state in chrome.

Reply:
Actually it did work like that at some point, but it's not working anymore. Investigating.

Reply:
Logged an issue here: #23598.